src/Illuminate/Foundation/helpers.php

```diff
@@ -722,11 +722,12 @@ function rescue(callable $callback, $rescue = null)
      * Resolve a service from the container.
      *
      * @param  string  $name
+     * @param  array  $params
      * @return mixed
      */
-    function resolve($name)
+    function resolve($name, array $params = [])
```

Comment:
Name this the same as in the `app` helper.

```suggestion
    function resolve($name, array $parameters = [])
```